app/controllers/api/learning_unit_successions_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  class LearningUnitSuccessionsController < ApiApplicationController
+    def index
+      cycle = Cycle.find(params[:cycle_id])
+      successions = LearningUnitSuccession.where(cycle:)
+      render json: successions, only: %i[predecessor_id successor_id]
```

Comment:
Won't you need the id of the successions? For instance, I would think that maybe to paint some edges or something like that. Is up to you.

Reply:
Not for the moment at least, we only need the connections and the direction of the connection